packages/node_modules/@webex/internal-plugin-feature/src/feature.js

```diff
@@ -37,6 +37,22 @@ const Feature = WebexPlugin.extend({
     return Promise.resolve(feature.value);
   },
 
+  /**
+   * Issues request to server to set a value for a feature toggle.
+   * @param {Object} envelope
+   * @returns {Promise} Refreshes the local device and resolves with the features endpoint's response.
+   */
+  handleFeatureUpdate(envelope) {
+    if (envelope && envelope.data) {
+      const feature = cloneDeep(envelope.data.featureToggle);
```

Comment:
why do we need to clone here? Do we mutate the object somewhere else?

Reply:
yeah when it gets added to the feature collection, it parses the object and mutates it. I see 2 events with the same id come through here. Not sure why there's 2 and the second one is the one that gets mutated

Reply:
i see where it's getting emitted twice: https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/internal-plugin-mercury/src/mercury.js#L442-L443

I think maybe server usually sends one or the other, but this looks like it sends both so it's the same object

Reply:
should i add a check in mercury.js, if the `namespace` === `data.eventType`, only emit 1 event? @mattnorris @taymoork2 @adamweeks

Reply:
@aimeex3 I think that should work!